utFileMonitor.py

```diff
@@ -0,0 +1,119 @@
+import sys
```

Comment:
RDK (c) must be present.

Reply:
add a module description on what it's for.

Reply:
https://github.com/rdkcentral/ut-raft/wiki/utFileMonitor-Manual -> to review and edit as required.